### PR TITLE
Update Image I/O page.

### PR DESCRIPTION
**Pull request by aaviram**
Update image I/O page with the relevant changes.

### CODE REVIEW

File: source/develop/release-management/features/image-upload.html.md [hunk 1]
```diff
@@ -1,232 +1,217 @@
...
-#### Goals
+### Goals
 
-1.  Upload VM Disk Image to Storage Domain
-2.  Upload ISO Image to ISO Domain
-3.  Download VM Disk Image from URL to Storage Domain
-4.  Download ISO Image from URL to ISO Domain
-5.  (soon) Provide a [REST] API to upload and/or perform random I/O on disk images on the storage
+1.  Upload VM images to oVirt Storage.
+3.  Upload VM images from a URL to oVirt Storage.
```
Comment (mureinik) [inline source/develop/release-management/features/image-upload.html.md]: What about 2?

File: source/develop/release-management/features/image-upload.html.md [hunk 2]
```diff
@@ -1,232 +1,217 @@
...
-#### Limitations
+### Limitations
 
 *   IE 10, Firefox 3.6, or Chrome 13 or greater are required to upload via Webadmin due to lack of necessary HTML5 APIs in older browsers. The UI will detect this and enable upload only if supported.
 
-#### User Experience
+### User Experience
 
-The upload action for a VM disk will be initiated and controlled in Webadmin by navigating to Storage > <file/block domain> > Disks and using the control buttons present; ISO upload will be under Storage > <ISO domain> > Images.
+The upload action for a VM disk is initiated and controlled in Webadmin by navigating to Storage > <file/block domain> > Disks, or Disks tab- And using the control buttons present; ISO upload will be under Storage > <ISO domain> > Images.
```
Comment (mureinik) [inline source/develop/release-management/features/image-upload.html.md]: Do the angular brackets render correctly in middleman?

File: source/develop/release-management/features/image-upload.html.md [hunk 3]
```diff
@@ -1,232 +1,217 @@
...
-#### Operation Sequence
+### Operation Sequence
 
-The following sequence diagram, described below, details the flow for a successful image upload from a machine logged into webadmin:
+The following sequence diagram, described below, details the flow for a successful image  transfer from a machine logged into webadmin:
 
 ![](Image_Upload-HTTP_Sequence_Diagram.png "Image_Upload-HTTP_Sequence_Diagram.png")
 
-(UMLet diagram source: <Media:Image_Upload_Sequence_Diagram.uxf.gz>
+Using the webadmin, when the user starts a transfer, the UI executes a TransferImageCommand on the backend to initiate the process. The command writes an ImageTransfer entity to the database to track the job and returns its ID tothe UI. The UI and command now both execute asynchronously: the UI polls the entity status using TransferImageCommand, while the backend initializing the transfer, (e.g. in upload, creates a disk/ISO if it is not already supplied, givesit a name, capacity, flags, etc.)
```
Comment (mureinik) [inline source/develop/release-management/features/image-upload.html.md]: givesit -> gives it

File: source/develop/release-management/features/image-upload.html.md (hunk 1 above)
Comment (aaviram) [inline source/develop/release-management/features/image-upload.html.md]: Done

File: source/develop/release-management/features/image-upload.html.md (hunk 2 above)
Comment (aaviram) [inline source/develop/release-management/features/image-upload.html.md]: It doesn't render to anything- just shows this text. I will **bold** it though, or mark it somehow.

File: source/develop/release-management/features/image-upload.html.md (hunk 3 above)
Comment (aaviram) [inline source/develop/release-management/features/image-upload.html.md]: Done

**Comment by aaviram** [comment]
@mureinik I rebased and changed the commit according to your review.

**Comment by mykaul** [comment]
@mureinik - any idea why can't I see it under http://www.ovirt.org/develop/release-management/features/storage/ , but I can access it directly via http://www.ovirt.org/develop/release-management/features/storage/image-upload/ ?

**Comment by aaviram** [comment]
The link should be ovirt-imageio (bug, should be fixed.), you can see the
page link to oVirt imageIO in http://www.ovirt.org/develop/
release-management/features/storage/.

**Pull request merged**